fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -1932,19 +1985,60 @@ protected void backfillResourceChangeLogDb2(TenantInfo ti) {
     /**
      * Backfill the RESOURCE_CHANGE_LOG table for all the resource types. Non-multi-tenant
      * implementation
+     *
      * @param adapter
      */
     private void doBackfill(IDatabaseAdapter adapter) {
         Set<String> resourceTypes = getResourceTypes();
 
-        for (String resourceTypeName: resourceTypes) {
+        for (String resourceTypeName : resourceTypes) {
             logger.info("Backfilling RESOURCE_CHANGE_LOG with " + resourceTypeName
-                + " resources for schema '" + schema.getSchemaName() + "'");
+                    + " resources for schema '" + schema.getSchemaName() + "'");
             BackfillResourceChangeLog backfill = new BackfillResourceChangeLog(schema.getSchemaName(), resourceTypeName);
             adapter.runStatement(backfill);
         }
     }
 
+    /**
+     * updates the vacuum settings for postgres.
+     */
+    public void updateVacuumSettings() {
+        if (dbType != DbType.POSTGRESQL) {
+            logger.severe("Updating the vacuum settings is only supported on postgres and the setting is for '" + dbType + "'");
+        }
```

Comment:
Return here, instead of continuing on with method?

Reply:
done, set a return.